app/javascript/components/gtl-view.jsx

```diff
@@ -110,7 +110,7 @@ const isCurrentControllerOrPolicies = (url) => {
   return splitUrl && (splitUrl[1] === ManageIQ.controller || splitUrl[2] === 'policies');
 };
 
-const EXPAND_TREES = ['treeview-policy_tree', 'treeview-savedreports_tree', 'treeview-widgets_tree'];
+const EXPAND_TREES = ['treeview-policy_tree', 'treeview-reports_tree'];
```

Comment:
Does this mean that we don't expand the savedreports tree? Why so?

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/components/gtl-view.jsx#L116 it will expand saved reports and widgets trees now and not expand the reports tree, because it is already expanded.